src/rsa/rsa.rs

```diff
@@ -43,6 +46,7 @@ fn parse_public_key(input: untrusted::Input)
 
 struct PositiveInteger {
     value: Option<*mut BIGNUM>,
+    bit_len: usize,
```

Comment:
Why add bit_len as a field, instead of implementing a method on `PositiveInteger` that calls `BN_num_bits()` or similar? It doesn't seem great to store this extra field here when it's value is only needed in during RSA signature verification. It's not a huge field but `RSAKeyPair` will eventually contain a bunch of these so it seems wasteful.